site/app/libraries/CustomParsedownEngine.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace app\libraries;
+use Aptoma\Twig\Extension\MarkdownEngineInterface;
+
+use Parsedown;
+
+/**
+ * CustomParsedownEngine.php
+ *
+ * Maps erusev/parsedown to Aptoma\Twig Markdown Extension
+ *
+ * @author Sébastien Lourseau <https://github.com/SebLours>
+ */
+class CustomParsedownEngine implements MarkdownEngineInterface
```

Comment:
Probably better to extend off `ParsedownEngine` and then you'd just have the constructor:
```
    public function __construct($instanceName = null) {
        parent::__construct($instanceName);
        $this->engine->setSafeMode(true);
    }
```

We don't really need the ability to configure this as I'm not sure we ever have a case where we would want to allow HTML in the markdown.